src/pause/140C70.c

```diff
@@ -1,17 +1,287 @@
 #include "common.h"
 
+#define pause_map_tab_places_description_string_id 0x1D00F2
+
 INCLUDE_ASM(s32, "pause/140C70", func_8024D930);
 
-INCLUDE_ASM(s32, "pause/140C70", pause_map_draw_border_arrows);
+void pause_map_draw_border_arrows(s32 arg0, s32 arg1, s32 arg2) {
+    s32 phi_a0;
```

Comment:
I think if you could name these args / temps that'd be ideal. Also is the `temp` one still needed after we matched it? If so, that's a bummer.

Reply:
`temp` was the thing that allowed us to match it. It wouldn't load it at the right place without it.